src/main/java/ladder/views/OutputView.java

```diff
@@ -50,4 +60,27 @@ private static String view(Point point, boolean last) {
 
         return POINT + EMPTY_SPACE;
     }
+
+    public static void print(LadderResults ladderResults, String name) {
+        System.out.println("\n실행 결과");
```

Comment:
System.lineSeparator()를 사용하면 OS마다 다른 줄바꿈 코드를 반영해줍니다.
```suggestion
        System.out.println(System.lineSeparator() + "실행 결과");
```